### ISSUE TITLE
ALT + Click not working under Linux; Right-click on part no longer functional

### ISSUE THREAD

**Issue by dertroglodyt** [linuxgurugamer/JanitorsCloset#1]
As stated above, it won't present a menu if I left-click + ALT.
The normal right-click only ever shows the expanded view of an item. (Not really an issue; maybe even better than stock behaviour ;-)
Right click also grabs part like left click.

Ubuntu 16.04, KSP_64, RSS+RO

**Comment by linuxgurugamer** [comment]
Alt is not the correct key on Linux.  I forget right now, but you need to use the MOD key, whichever it is:
http://wiki.kerbalspaceprogram.com/wiki/Key_bindings

Linux: KSP 1.1+: Right-Shift (e. g. Right-Shift+L) Don't confuse this with Left-Shift, which will fire your engines! KSP 1.0.5 and earlier uses a different key, which can cause issues in some window managers when playing in windowed mode.

**Comment by dertroglodyt** [comment]
I know about right shift. Nontheless not functioning.

Am 14.09.2016 22:25 schrieb "linuxgurugamer" <notifications@github.com>:

**Comment by linuxgurugamer** [comment]
This will have to wait until after the 1.2 release, I'll work on it then, will need to get a linux system up and running to test

**Comment by linuxgurugamer** [comment]
Please try again with current release

**Comment by dertroglodyt** [comment]
Seems to work beautifully now. Thanks a lot.
One big wish: A confirmation dialog for "unblock". It's very easy to misclick on a 4k monitor.

**Comment by linuxgurugamer** [comment]
Open a new issue as a Feature Request for that
